### PR TITLE
Add new TDTD types, remove deathOfOwner

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15263

*Description of changes:*
- Added new registration types: TRANS_ADMIN, TRANS_AFFIDAVIT, TRANS_WILL
- Updated validation for these new TDTD types
- Updated tests
- Updated postman collection to contain a transfer for each of the new transfer types

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: mhr_api/tests/unit/utils/test_registration_validator.py [hunk 1]
```diff
@@ -852,6 +852,101 @@ def test_validate_transfer_death(session, desc, valid, party_type1, party_type2,
         if message_content:
             assert error_msg.find(message_content) != -1
 
+@pytest.mark.parametrize('desc,valid,party_type1,party_type2,text,add_group,message_content', TEST_TRANSFER_DEATH_DATA)
```
Comment (doug-lovett) [inline mhr_api/tests/unit/utils/test_registration_validator.py:855]: Nice, thanks for adding the new tests.

**Review by doug-lovett — CHANGES_REQUESTED** [review]
Looks great, just a few comments:
- Increment the api version
- Update the Postman collection to add an example for each of the 3 new registration types, and remove deathOfOwner from the existing transfer examples. Divya will use the Postman examples as a source for testing the API.
- In the future, typically UI changes are captured in a separate PR.
- The remaining request data validation updates for transfer due to death registrations are now covered in a separate ticket.

**Review by doug-lovett — APPROVED** [review]
Thanks for making the changes, looks good.

**Review by cameron-eyds — APPROVED** [review]
LGTM!

**Review by dimak1 — APPROVED** [review]
Nice work @owillborn!

**Pull request merged**